### PR TITLE
Create Simple JSON format for Hydra V2

### PR DESCRIPTION
**Pull request by lewisjkl**

### CODE REVIEW

File: ingest/src/main/scala/hydra.ingest/http/IngestionEndpoint.scala [hunk 1]
```diff
@@ -117,9 +117,8 @@ class IngestionEndpoint[F[_]: Futurable](
         optionalHeaderValueByName(correlationIdHeader) { cIdOpt =>
           entity(as[V2IngestRequest]) { reqMaybeHeader =>
             val req = cIdOpt match {
-              case Some(id) => reqMaybeHeader.copy(reqMaybeHeader.keyPayload, reqMaybeHeader.valPayload,
-                reqMaybeHeader.validationStrategy, Some(Headers.fromSeq(List(Header.apply(correlationIdHeader, id)))))
-              case _ => reqMaybeHeader.copy(reqMaybeHeader.keyPayload, reqMaybeHeader.valPayload, reqMaybeHeader.validationStrategy, None)
+              case Some(id) => reqMaybeHeader.copy(headers = Some(Headers.fromSeq(List(Header.apply(correlationIdHeader, id)))))
```
Comment (w-aldrich) [inline ingest/src/main/scala/hydra.ingest/http/IngestionEndpoint.scala:120]: This is a lot cleaner thank you.

**Review by w-aldrich — APPROVED** [review]

**Comment by w-aldrich** [comment]
Does this pass e2e testing?

**Comment by lewisjkl** [comment]
> Does this pass e2e testing? If it does, everything looks good to me.

Just ran them and all pass. Thanks for the reminder.

**Comment by lewisjkl** [comment]
All tests pass locally. Going to merge now.

**Pull request merged**